src/tailscale/status.rs

```diff
@@ -22,23 +22,23 @@ pub struct Status {
     user: HashMap<String, User>,
 }
 
-pub fn get_status() -> Result<Status, Box<dyn std::error::Error>> {
```

Comment:
We already have the module name as `status`. Clippy warns against this redundancy, since it's going to be used like `status::get` anyway. Pretty explicit.